routers/api/v2/tokens.go

```diff
@@ -53,9 +47,17 @@ func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
 		}
 	}
 
-	token, err := r.authorizer.CreateServiceToken(req.Owner, parsedURIs, req.ExpiresAt)
+	userToken := r.GetAuthToken(ctx)
+	userID, err := r.authorizer.GetUserIdFromToken(userToken)
 	if err != nil {
```

Comment:
Could you update this to handle the different types of errors`GetUserIdFromToken` can return? Something like this: https://github.com/unicsmcr/hs_auth/blob/master/routers/api/v2/teams.go#L48-L61